nautobot/dcim/templates/dcim/controllermanageddevicegroup_retrieve.html

```diff
@@ -1,6 +1,50 @@
 {% extends 'generic/object_retrieve.html' %}
 {% load helpers %}
 
+{% block extra_nav_tabs %}
+    {# TODO Currently going off of wireless_networks and radio_profiles count, but may switch to capabilities once implemented. #}
```

Comment:
Still a TODO or can we do this now?

Reply:
I wanted to get feedback on this one... Our standard is to use count, and since these are directly on the model, I don't have to do this the same way as we do for Controllers. Rendering the tabs when there is no data is not the common pattern.